src/client/tile.cpp

```diff
@@ -1811,6 +1811,23 @@ bool TextureSource::generateImagePart(std::string part_of_name,
 	return true;
 }
 
+/*
+	Calculate the color of a single pixel drawn on top of another pixel.
+
+	This is a little more complicated than just video::SColor::getInterpolated
+	because getInterpolated does not handle alpha correctly.  For example, a
+	pixel with alpha=64 drawn atop a pixel with alpha=128 should yield a
+	pixel with alpha=160, while getInterpolated would yield alpha=96.
+*/
+static video::SColor blit_pixel(video::SColor src_c, video::SColor dst_c, u32 ratio)
```

Comment:
should be `inline` and pass the SColor arguments by reference

Reply:
FYI: SColor is `u32`, hence references are OK but not really needed.